.github/actions/zombienet-v1/action.yml

```diff
@@ -0,0 +1,53 @@
+name: "Zombienet test v1"
+inputs:
+  test:
+    description: "test definition (zndsl file)"
+    required: true
+  local-dir:
+    description: "Path to the directory tha contains the test file (.zndsl)"
+    required: true
+  concurrency:
+    description: "Concurrency to spawn nodes"
+    default: 4
+    required: false
+
+runs:
+  using: "composite"
+  steps:
+    - name: k8s_auth
+      shell: bash
+      run: |
+        . /home/nonroot/zombie-net/scripts/ci/run-test-local-env-manager.sh
+        k8s_auth
+
+    - name: common_vars
+      shell: bash
+      run: |
+        echo "Vars"
+        echo "ZOMBIENET_INTEGRATION_TEST_IMAGE: $ZOMBIENET_INTEGRATION_TEST_IMAGE"
+        echo "COL_IMAGE: $COL_IMAGE"
+        echo "Inputs"
+        echo "test: ${{ inputs.test}}"
+        echo "local-dir: ${{ inputs.local-dir }}"
+        echo "concurrency: ${{ inputs.concurrency }}"
+
+    - name: zombie_test
+      shell: bash
+      run: |
+        /home/nonroot/zombie-net/scripts/ci/run-test-local-env-manager.sh \
+          --local-dir="${{ inputs.local-dir }}" \
+          --concurrency=${{ inputs.concurrency }} \
+          --test="${{ inputs.test }}"
+      continue-on-error: true
+
+    - name: upload_logs
+      uses: actions/upload-artifact@v4
+      with:
+        name: zombienet-logs-${{ github.job }}-${{ github.sha }}
+        path: |
+          /tmp/zombie*/logs/*
+      continue-on-error: true
+
+    - name: report_failure
+      if: steps.zombie_test.outcome != 'success'
+      run: exit 1
```

Comment:
You could add `if: always()` to the `upload_logs` job and remove this one.

Reply:
But in that case we report the error too?

Reply:
Yes, it'll report anyway

Reply:
awesome, thanks!